test/hotspot/jtreg/runtime/cds/appcds/TestParallelGCWithCDS.java

```diff
@@ -125,31 +125,22 @@ static void test(boolean dumpWithParallel, boolean execWithParallel, boolean use
         if (!dumpWithParallel && execWithParallel) {
             // We dumped with G1, so we have an archived heap. At exec time, try to load them into
             // a small ParallelGC heap that may be too small.
-            String[] sizes = {
-                "4m",   // usually this will success load the archived heap
-                "2m",   // usually this will fail to load the archived heap, but app can launch
-                        // or fail with "GC triggered before VM initialization completed"
-                "1m"    // usually this will cause VM launch to fail with "Too small maximum heap"
-            };
-            for (String sz : sizes) {
-                String xmx = "-Xmx" + sz;
-                System.out.println("=======\n" + n + ". Exec with " + execGC + " " + xmx);
-                out = TestCommon.exec(helloJar,
-                                      execGC,
-                                      small1,
-                                      small2,
-                                      xmx,
-                                      coops,
-                                      "-Xlog:cds",
-                                      "Hello");
-                if (out.getExitValue() == 0) {
-                    out.shouldContain(HELLO);
-                    out.shouldNotContain(errMsg);
-                } else {
-                    out.shouldNotHaveFatalError();
-                }
-                n++;
+            System.out.println("=======\n" + n + ". Exec with " + execGC);
```

Comment:
Since you got rid of the `for` loop, I think it can be simplified without using the `n` variable.
```suggestion
           System.out.println("2. Exec with " + execGC);
```